examples/bundle-webpack/src/components/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
```

Comment:
If we're bundling this with webpack and stage-0 + react, files are already always running in strict mode.